src/configuration.rs

```diff
@@ -109,6 +109,8 @@ pub enum ServiceType {
 pub struct PluginConfiguration {
     pub services: HashMap<String, Service>,
     pub action_sets: Vec<ActionSet>,
+    #[serde(default)]
+    pub metrics_enabled: Option<bool>,
```

Comment:
Do we need the `Option` here? Can't we have serde default to [`bool`'s impl of `Default`, i.e. `false`](https://doc.rust-lang.org/std/primitive.bool.html#method.default)?